### PR TITLE
New UI Test Suite

### PR DESCRIPTION
**Pull request by Winterhart**
## Description

 - Addition of a UI test suite
 - First github workflow with playwright
 - New folder for UI testing
 - UI test on `DataTable` and `AreaChart`
 - Addition of `aria-label` on button increasing our accessibility score
 - Addition of an `id` `tooltip` on all chart tooltip

### Demo in `VSCode`

613-demo-vscode-playwright

### Before `aria-label`
Screenshot 2023-02-03 at 9 43 14 AM

### After `aria-label`
Screenshot 2023-02-03 at 10 00 55 AM

The accessibility ramp-up to `97` for `DataTable` page

### Checklist
- [x] For UI or styling changes, I have added a screenshot or gif showing before & after
- [x] I have added a [changeset](https://github.com/evidence-dev/evidence/blob/main/CONTRIBUTING.md#adding-a-changeset)

### CODE REVIEW

File: tests/playwright.config.ts [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+  use: {
+    /* Maximum time each action such as `click()` can take. Defaults to 0 (no limit). */
+    actionTimeout: 0,
+    /* Base URL to use in actions like `await page.goto('/')`. */
+    baseURL: process.env.BASE_URL ? process.env.BASE_URL : 'http://localhost:3000',
+
+    /* Collect trace when retrying the failed test. See https://playwright.dev/docs/trace-viewer */
+    trace: 'on-first-retry',
+  },
+
```
Comment (ud3sh) [inline tests/playwright.config.ts:40]: Did your tests run well on all these browsers without issues btw?

**Comment by ud3sh** [comment]
@Winterhart  Should we create an issue for sorting out best practices for coming up resilient selectors for this project (test-ids vs css etc) ?

File: tests/playwright.config.ts (hunk 1 above)
Comment (Winterhart) [reply tests/playwright.config.ts:40]: We got them running in Github actions on the "different browser package"

**Comment by Winterhart** [comment]
Yes, we will have to find a common strategy for our UI testing suite.

**Pull request merged**